lldb/source/Plugins/Process/minidump/MinidumpParser.cpp

```diff
@@ -114,7 +114,13 @@ MinidumpParser::GetThreadContextWow64(const minidump::Thread &td) {
   // process rather than the 32-bit "guest" process we care about.  In this
   // case, we can get the 32-bit CONTEXT from the TEB (Thread Environment
   // Block) of the 64-bit process.
-  auto teb_mem = GetMemory(td.EnvironmentBlock, sizeof(TEB64));
+  auto teb_mem_maybe = GetMemory(td.EnvironmentBlock, sizeof(TEB64));
+  if (!teb_mem_maybe) {
+    llvm::consumeError(teb_mem_maybe.takeError());
```

Comment:
Should we use `LLDB_LOG_ERROR` here? Same question below.

Reply:
I personally don't have a lot of context on if this should be an error (WOW even when I was at MSFT wasn't a common talking point). But I'm going to make the change